Entities/Industry/CTFShops/Quarters/Quarters.as

```diff
@@ -277,6 +276,7 @@ void onAttach(CBlob@ this, CBlob@ attached, AttachmentPoint@ attachedPoint)
 		if (head !is null)
 		{
 			texName = head.getFilename();
+			frame = head.getFrame();
```

Comment:
`head.getFrame();` returns the frame of the spritesheet currently used in the animation. If I idle the spritesheet frame is offset + 0. However, if I (for example) slash the moment I enter the bed the spritesheet frame is offset + 1. Then `anim.AddFrame(frame + 2)` adds the spritesheet frame at offset + 3 (a blank frame).

Replace line 279:
```
Animation@ animation = head.getAnimation('default');
if (animation !is null)
{
  frame = animation.getFrame(2);
}
```

Replace line 299:
```
anim.AddFrame(frame);
```